src/config/rt.rs

```diff
@@ -252,6 +252,12 @@ pub struct RtcServe {
     pub proxies: Option<Vec<ConfigOptsProxy>>,
     /// Whether to disable auto-reload of the web page when a build completes.
     pub no_autoreload: bool,
+    /// The path to the private key. TLS is activated if both are set.
+    pub tls_private_key_path: Option<String>,
+    /// The path to the public key. TLS is activated if both are set.
+    pub tls_public_key_path: Option<String>,
```

Comment:
- Technically it is not a public key which is needed here, it is a TLS certificate which is needed. Looks like the example does indeed have the certificate specified, so that it good. Let's fix the nomenclature though. Instead of public key, it should be `cert`: `tls_cert_path`.
- Also, to match industry patterns, let's just keep this as simple as possible and rename `tls_private_key_path` to just `tls_key_path`.

This will results in the 2 new config params being: `tls_key_path` & `tls_cert_path`.

Reply:
Sure, that's the easy part :)